react/getting_started_with_react/react_components.md

```diff
@@ -84,7 +84,7 @@ And just like that, you've successfully imported and used your first custom-made
 <div class="lesson-content__panel" markdown="1">
 
 1.  It's time to create some new components! Use the same project, but play around with it, try displaying something like your favorite food.
-    *   While components normally get exported as defaults, try using some named exports instead of default exports. If unsure of how to do this, consult your best friend - [MDN its export documentation](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#description)
+    *   While components normally get exported as defaults, try using some named exports instead of default exports. If unsure of how to do this, consult your best friend - [MDN-its export documentation](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#description)
```

Comment:
```suggestion
    *   While components normally get exported as defaults, try using some named exports instead of default exports. If unsure of how to do this, consult your best friend: [the MDN documentation about export statements](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#description)
```

I suggested a change that sounds more like a sentence to me. What do you think?
If you like it you can click `commit suggestion` if you want instead of rewriting it yourself and pushing changes to the repo by the way.

Reply:
Looks good to me!